### DISCUSSION TITLE
Tailwind animation: animate-pulse

### DISCUSSION THREAD

**Discussion by mjapadilla** [tailwindlabs/tailwindcss#11307] [Help]
Tailwind 3.3.2

Using animate-pulse it should be animating opacity  from 50% to 100% but instead it animate the size
From thtis
![image](https://github.com/tailwindlabs/tailwindcss/assets/56869280/94846d37-f7af-4729-8b78-fe7069362a0f)
To this
![image](https://github.com/tailwindlabs/tailwindcss/assets/56869280/b9fac106-6a5b-4f7b-92ef-46ba176f00b1)

**Comment by wongjn** [comment]
Please consider reproducing the undesired behavior in a [Tailwind Play](https://play.tailwindcss.com).

**Reply by janhesters** [reply]
I have the same behavior 🤔 

It's even in the docs for me lol.

<img width="781" alt="Screenshot 2024-09-01 at 15 04 13" src="https://github.com/user-attachments/assets/503a31d9-4cf6-487f-af1e-5e9614e81be9">

**Comment by aumirza** [comment]
it also happening with me

  "tailwindcss": "^3.2.4"

https://github.com/user-attachments/assets/c7cbe91b-93b1-46f6-a345-aa7f1da48058

## Solution i used

Add this to your tailwind config file
          `
                 extend: {
                    animation: {
                      pulse: "pulseOpacity 1.5s cubic-bezier(0.4, 0, 0.6, 1) infinite",
                    },
                    keyframes: {
                      pulseOpacity: {
                        "0%, 100%": { opacity: 1 },
                        "50%": { opacity: 0.5 },
                      },
                    },
                  },
          `
